src/benchmarks/micro/corefx/System.Text.Json/Utf8JsonReader/Utf8JsonReader.MultiSegment.Tests.cs

```diff
@@ -0,0 +1,283 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Can the file have comments in the name since all these tests are specific to payloads with comments?